src/main/java/com/faforever/client/api/dto/AbstractEntity.java

```diff
@@ -0,0 +1,18 @@
+package com.faforever.client.api.dto;
+
+import com.github.jasminb.jsonapi.annotations.Id;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.OffsetDateTime;
+
+@Getter
+@Setter
+@EqualsAndHashCode(of = "id")
+public abstract class AbstractEntity {
+  @Id
+  protected String id;
+  protected OffsetDateTime createTime;
+  protected OffsetDateTime updateTime;
+}
```

Comment:
There seem to be a bunch of other classes that maybe should make use of this to stay consistent with API code? Or do you want to change those when they get touched anyway? Maybe we should have an issue were all classes that should use this eventually are listed.

Reply:
Well in the long run we wanna change to java-commons dtos but that is a BIIIGGGG task

Reply:
I just copied the calsses from there. I would like to see more usage of AbstractEntity  in the future yes... But chaning beyond this seems out of scope of the PR

Reply:
Ok